scope/scope.go

```diff
@@ -0,0 +1,70 @@
+package scope
+
+import (
+	"fmt"
+
+	"github.com/taskcluster/taskcluster-cli/extpoints"
+	"github.com/taskcluster/taskcluster-client-go"
+	"github.com/taskcluster/taskcluster-client-go/auth"
+)
+
+func init() {
+	extpoints.Register("scope", scope{})
+}
+
+type scope struct{}
+
+func (scope) ConfigOptions() map[string]extpoints.ConfigOption {
+	return nil
+}
+
+func (scope) Summary() string {
+	return "Expand the given scope set."
+}
+
+func usage() string {
+	return `Usage:
+  taskcluster scope expand <scope>...
+
+### Expand
+This command returns an expanded copy of the given scope set, with scopes
+implied by any roles included. The given scope set is specified as a space
+separated list of scopes.
+`
+}
+
+func (scope) Usage() string {
+	return usage()
+}
+
+func (scope) Execute(context extpoints.Context) bool {
+	argv := context.Arguments
+
+	authCreds := tcclient.Credentials{}
+	myAuth := auth.New(&authCreds)
```

Comment:
nit:
`a := auth.New(&tcclient.Credentials{})`

And let's not pass it as argument to `expandScope` we can just create it inside `expandScope`..